### PR TITLE
Bug 1414107: Move breadcrumbs to left column

### PR DESCRIPTION
**Pull request by stephaniehobson**
- remove crumbs from document-head
- add crumbs to left column
- change from rdf to schema.org encoding on crumbs
- add intelligent break points to long page titles
- tweak flow of document-title and document-actions on mobile and tablet
- update look of crumbs
    - including larger hit area on mobile and tablet

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@schalkneethling waiting on some help to fix the breaking test but you could review the non-test code if you have time.

File: kuma/static/styles/components/wiki/crumbs.scss [hunk 1]
```diff
@@ -1,47 +1,73 @@
...
 
     li {
-        @include bidi-style(margin-right, ($grid-spacing / 2), margin-left, 0);
-        display: none;
+        display: inline;
     }
 
-    li:nth-last-child(2) {
-        display: block;
+    a { // @include reverse-link-decoration has buggy behaviour in .crumbs for some reason
```
Comment (schalkneethling) [inline kuma/static/styles/components/wiki/crumbs.scss]: Nit: CSS style comments?

**Comment by schalkneethling** [comment]
By default, I cannot see the crumbs above the quick links, but I can clearly see it taking up space:

screen shot 2017-11-08 at 13 47 59

If you either remove `position: relative;` or `background-color: #fff;` then they become visible.

screen shot 2017-11-08 at 13 48 16

**Comment by schalkneethling** [comment]
At phone viewports the bottom margin below the crumbs are lost:

screen shot 2017-11-08 at 13 52 33

**Comment by schalkneethling** [comment]
@stephaniehobson Am I correct that the crumbs will now only ever be one level back? i.e. On a page such as `docs/Web/JavaScript/Reference/Statements`, the breadcrumb will now be just `< JavaScript reference` and not say `< JavaScript reference < Statements` or `< JavaScript < JavaScript reference`

**Comment by schalkneethling** [comment]
Ignore those comments, I have removed them. I keep forgetting to run `gulp` after fetching a new branch :-/

**Comment by schalkneethling** [comment]
I really like the larger hit area on mobile but, I do wonder whether we should scale it down a bit to `10px`

screen shot 2017-11-08 at 14 18 04

as apposed to the current `15px`

screen shot 2017-11-08 at 14 17 42

**Comment by jwhitlock** [comment]
I've updated the tests for the new behavior:

- There are no breadcrumbs if there is no left sidebar. A quick links section (usually KumaScript), a zone navigation section, or editorial / technical review flags can add a left sidebar.
- Even with a left sidebar, root-level documents don't get breadcrumbs.

I tried to keep the test changes minimal, which meant not converting a Django test to pytest style.

**Pull request merged**